setup.py

```diff
@@ -46,7 +46,7 @@ def exec_file(path_segments, name):
     py_modules=["canonicaljson"],
     description="Canonical JSON",
     install_requires=[
-        "simplejson>=3.6.5",
+        "simplejson>=3.6.5; platform_python_implementation == 'CPython'",
```

Comment:
any idea what version of setuptools this requires, and hence which OSes we are excluding here?